### ISSUE TITLE
Docs on testing?

### ISSUE THREAD

**Issue by Empact** [tardate/authlogic_rpx#14]
I'm having trouble testing due to the time-sensitive nature of the RPX tokens, and not knowing how to generate one myself or stub out the necessary interactions internally.  Could we have a few lines on this in the docs?

**Comment by tardate** [comment]
you're right - there's got to be a better way, and more coverage in the docs is a good idea.

Check out the tests included for an example of testing with mocks (e.g. http://github.com/tardate/authlogic_rpx/blob/master/test/integration/basic_authentication_and_registration_test.rb ). To test with the RPX service for real, I still just do it manually.
